ecs-cli/modules/cli/regcreds/create_task_execution_role.go

```diff
@@ -119,3 +120,15 @@ func attachRolePolicies(secretPolicyARN, roleName, region string, client iamClie
 
 	return nil
 }
+
+func convertToIAMTags(tags map[string]*string) []*iam.Tag {
+	var iamTags []*iam.Tag
+	for key, value := range tags {
+		iamTags = append(iamTags, &iam.Tag{
+			Key:   aws.String(key),
+			Value: value,
```

Comment:
Is it valid to have a `nil` value for a tag? If so, can we add a case for it to `TestCreateTaskExecutionRoleWithTags` ?

Reply:
Docs say its a required field: https://docs.aws.amazon.com/sdk-for-go/api/service/iam/#Tag

You can pass an empty string though, which my code handles correctly:

```
$ ecs-cli registry-creds up --role-name catsss --tags cat=,meow= reg-creds.yml
INFO[0000] Processing credentials for registry my-registry.example.com...
INFO[0000] Existing credential secret found, using arn:aws:secretsmanager:us-west-1:REDACTED:secret:amazon-ecs-cli-setup-my-registry.example.com-TIxAwp
INFO[0000] Creating resources for task execution role catsss...
INFO[0000] Created new task execution role arn:aws:iam::REDACTED:role/catsss
INFO[0000] Created new task execution role policy arn:aws:iam::REDACTED:policy/amazon-ecs-cli-setup-catsss-policy-20190213T184615Z
INFO[0001] Attached AWS managed policy arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy to role catsss
INFO[0001] Attached new policy arn:aws:iam::REDACTED:policy/amazon-ecs-cli-setup-catsss-policy-20190213T184615Z to role catsss
INFO[0001] Writing registry credential output to new file stuffnthings/bugbash/compose3/ecs-registry-creds_20190213T184615Z.yml
INFO[0001]
If your input file contains sensitive information, make sure that you delete it after use.
```

(I checked that the tags had been set in the Console)

Reply:
Do we want to log info about what tags are being applied to what resources?